test/system/500-networking.bats

```diff
@@ -677,8 +677,8 @@ EOF
 @test "podman run port forward range" {
     for netmode in bridge slirp4netns:port_handler=slirp4netns slirp4netns:port_handler=rootlesskit; do
         local range=$(random_free_port_range 3)
-        local port="${test%-*}"
-        local end_port="${test#-*}"
+        local port="${range%-*}"
+        local end_port="${range#-*}"
```

Comment:
@Luap99 PTAL. The obvious question is, how did this ever work?

Reply:
It simply did not work. We just never looped in `for port`.

But this is not even the actual problem I was reporting. The test picked `5354-5356` which is wrong, 5355 is always blocked. So the free port logic has a bug as well.